packages/browser-extension/src/scripts/dialog.js

```diff
@@ -90,7 +92,18 @@ const reportDialogHtml = `
               tabindex="0">
               ${browser.i18n.getMessage('reportDialog_popupShowUpOption')}
             </report-dialog-radio>
+            <report-dialog-radio 
+            aria-checked="false" 
```

Comment:
Please, check the tabulation here because it's not aligned well.

Reply:
done